pkg/utils/utils.go

```diff
@@ -553,6 +553,11 @@ func GetMCEComponents(mch *operatorsv1.MultiClusterHub) []mcev1.ComponentConfig
 			config = append(config, mcev1.ComponentConfig{Name: n, Enabled: mch.Enabled(n)})
 		}
 	}
+	if mch.Spec.DisableHubSelfManagement {
```

Comment:
Am I correct in reading this that there are now 2 places to disable hub self-management? Can we have the case where we have set `MCELocalCluster` `enabled` set to false without `disableHubSelfManagement` being set?

Reply:
When ACM is installed, `disableHubSelfManagement` will be used to control `MCELocalCluster`. If you change `MCELocalCluster` with ACM installed, mch-o should change it back. I realized I haven't tested this yet and will today.